docker/datahub-gms/start.sh

```diff
@@ -64,7 +66,7 @@ COMMON="
     $PROMETHEUS_AGENT \
     -jar /jetty-runner.jar \
     --jar jetty-util.jar \
-    --jar jetty-jmx.jar \
+    --jar jetty-jmx.jar --classes ${AUTH_RESOURCES_DIR} \
```

Comment:
This won't break is AUTH_RESOURCES_DIR is empty right? Please confirm!

Reply:
Confirmed. It wouldn't break.